todo_app/app.py

```diff
@@ -1,5 +1,5 @@
 from flask import Flask, redirect, request, url_for, render_template
-from todo_app.data import trello_items
+from todo_app.data import mongo_client
```

Comment:
Nice, the fact this file has had such simple changes is a really strong testament to the sensible design you chose for the trello_items service all that time ago! 😄